python/amico/core.py

```diff
@@ -51,8 +52,9 @@ def __init__( self, study_path, subject ) :
 
         self.set_config('peaks_filename', None)
         self.set_config('doNormalizeSignal', True)
+        self.set_config('doKeepb0Intact', False)  # does change b0 images in the predicted signal
         self.set_config('doComputeNRMSE', False)
-
+        self.set_config('doSaveCorrectedDWI', False)
```

Comment:
I'm ok with the, the whole project (especially the python version) is still work-in-progress. We just need to be sure that it does not break the processing with other models, like referring to fields that are specific to the WaterFree model